src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -835,9 +835,19 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
-                        ->booleanNode('disable_not_compromised_password')
-                            ->defaultFalse()
-                            ->info('Disable NotCompromisedPassword Validator: the value will always be valid.')
+                        ->arrayNode('not_compromised_password')
+                            ->addDefaultsIfNotSet()
+                            ->info('not-compromised password validator configuration')
+                            ->children()
+                                ->booleanNode('enabled')
```

Comment:
can be replaced with `canBeEnabled`?

Reply:
IMHO enabled is better, as the validator is enabled and ready to be used by default. Also @stof suggested this naming.

Reply:
so `canBeDisabled` then? These methods add the `enabled` child node ...

Reply:
I agree with @ro0NL here.

Reply:
Before I merge, do we want to switch to `canBeDisabled()`?

Reply:
When using `canBeDisabled` instead of `addDefaultsIfNotSet`, the tests still pass and the configuration works. However, we'd still need the booleanNode configuration to add the info what enabling/disabling does.

What do you think? For me both options are fine. Thanks!

Reply:
I think using `canBeDisabled` makes it more similar to the other configuration settings we have.

Reply:
Adapted. Do you think that the default value and info text should still be provided to make things clearer for users?